libslide/src/parser/expression_pattern_parser.rs

```diff
@@ -63,6 +65,10 @@ impl Parser<InternedExprPat> for ExpressionPatternParser {
     fn parse_any_pattern(&mut self, name: String, span: Span) -> Self::Expr {
         intern_expr_pat!(ExprPat::AnyPat(name), span)
     }
+
+    fn has_stmt_break(&mut self) -> bool {
+        false
+    }
 }
```

Comment:
why do you need a function here if it always ret false

Reply:
Because declared has_stmt_break on the parser trait